### DISCUSSION TITLE
How to pass pandoc arguments in yaml

### DISCUSSION THREAD

**Discussion by harrelfe** [quarto-dev/quarto-cli#630] [Q&A] [answered]
With Rmarkdown I could have `pandoc_args: ["--number-offset=21,0"]` in the yaml header.  I haven't been able to figure out how to specify this with `Quarto`.  I assume the command is to be indented under `execute:` but I haven't been able to get it to work.

**Comment by jjallaire** [comment, marked as answer]
All Pandoc command line args have YAML equivalents (all of which work in Quarto). The pandoc docs provide this mapping here: https://pandoc.org/MANUAL.html#defaults-files

More generally, for any given output format there is a reference guide that combines Pandoc metadata, Pandoc "defaults", and Quarto-specific options. For example, here are the PDF options: https://quarto.org/docs/reference/formats/pdf.html

**Reply by harrelfe** [reply]
Thanks JJ - I see now -  `number-offset: 12` goes under `html:`.
